docs/reference/upgrade/reindex_upgrade.asciidoc

```diff
@@ -115,6 +115,9 @@ indices from the previous major version to be upgraded to the
 current major version. Skipping a major version means that you must
 resolve any backward compatibility issues yourself.
 
+{es} does not support downgrades or forward compatibility across major versions.
```

Comment:
While it's true we don't support downgrades, I wonder if it dilutes the message here?

```suggestion
{es} does not support forward compatibility across major versions.
```